docs/faq.md

```diff
@@ -82,3 +82,18 @@ You can find the details about how do we calculate these metrics [here](https://
 
 In the overview blade of Azure SignalR Service resources, we have already chosen the approperate aggregation type for you. And if you go to the Metrics blade, you can
 take the aggregation type [here](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftsignalrservicesignalr) as a reference.
+
+<a name="service-mode"></a>
+## What is the meaning of service mode `Default`/`Serverless`/`Classic`? How can I choose?
+
+Modes:
+* `Default` mode **require** hub server. When a hub without any available server connection, client connection will be reject.

```

Comment:
>require [](start = 19, length = 7)

`require` => `requires`
`When there is no server connection available for the hub, the client tries to connect to this hub fails.`?